pkgs/top-level/all-packages.nix

```diff
@@ -20208,6 +20208,8 @@ in
   inherit (gnome3) evince;
   evolution-data-server = gnome3.evolution-data-server;
   evolution-ews = callPackage ../applications/networking/mailreaders/evolution/evolution-ews { };
+  evolution = callPackage ../applications/networking/mailreaders/evolution/evolution { };
+  evolutionBundle = callPackage ../applications/networking/mailreaders/evolution/evolution/wrapper.nix { };
```

Comment:
This is non-standard naming that is not used anywhere. The `-with-plugins` is at least almost an idiom.

Reply:
If we are to talk about naming: the problem is that this is rather an evolution-data-server-with-plugins. Where evolution somehow counts as a plugin. an evolutionWithPlugins which can possibly *not* contain evolution but only evolution-data-server would be somewhat strange. So I welcome better naming suggestions which can convey that.